guides/source/active_record_migrations.md

```diff
@@ -46,87 +55,63 @@ class CreateProducts < ActiveRecord::Migration[7.2]
-passed to `direction.up` and `direction.down` respectively.
-
-Alternatively, you can use `up` and `down` instead of `change`:
+We define the change that we want to happen moving forward in time. Before this
+migration is run, there will be no table. After it is run, the table will exist.
+Active Record knows how to reverse this migration as well; if we roll this
+migration back, it will remove the table. You can read more about rolling back
+migrations [here](#rolling-back).
 
-```ruby
-class ChangeProductsPrice < ActiveRecord::Migration[7.2]
-  def up
-    change_table :products do |t|
-      t.change :price, :string
-    end
-  end
-
-  def down
-    change_table :products do |t|
-      t.change :price, :integer
-    end
-  end
-end
-```
+After defining the change that we want to occur moving forward in time, it's
+essential to consider the reversibility of the migration. While Active Record
+can manage the forward progression of the migration, ensuring the creation of
+the table, the concept of reversibility becomes crucial. With reversible
+migrations, not only does the migration create the table when applied, but it
+also enables smooth rollback functionality. In case of reverting the migration,
+Active Record intelligently handles the removal of the table, maintaining
+database consistency throughout the process. You can read more about reversing
+migrations [here](#using-reversible).
 
-INFO: More on [`reversible`](#using-reversible) later.
 
 Generating Migrations
 ----------------------
 
 ### Creating a Standalone Migration
 
 Migrations are stored as files in the `db/migrate` directory, one for each
-migration class. The name of the file is of the form
-`YYYYMMDDHHMMSS_create_products.rb`, that is to say a UTC timestamp
-identifying the migration followed by an underscore followed by the name
-of the migration. The name of the migration class (CamelCased version)
-should match the latter part of the file name. For example
-`20080906120000_create_products.rb` should define class `CreateProducts` and
-`20080906120001_add_details_to_products.rb` should define
+migration class.
+
+The name of the file is of the form `YYYYMMDDHHMMSS_create_products.rb`, that is
+to say a UTC timestamp identifying the migration followed by an underscore
+followed by the name of the migration. The name of the migration class
+(CamelCased version) should match the latter part of the file name. For example,
+`20240502100843_create_products.rb` should define class `CreateProducts` and
+`20240502101659_add_details_to_products.rb` should define class
 `AddDetailsToProducts`. Rails uses this timestamp to determine which migration
 should be run and in what order, so if you're copying a migration from another
 application or generate a file yourself, be aware of its position in the order.
 
-Of course, calculating timestamps is no fun, so Active Record provides a
+Of course, calculating timestamps manually is tedious, so Active Record provides a
```

Comment:
This example feels out of place. What is the connection between timestamps and part numbers here, or what is this solving for exactly?